docs/fxa-engineering/fxa-sub-platform.md

```diff
@@ -0,0 +1,105 @@
+
+The `fxa-payments-server` needs the Stripe public key and communicates with the `fxa-auth-server` that requires a Stripe private key. Configuration details can be found below.
+
+### Configuration
+
+You will need to add a `subscriptions.stripeApiKey` field in `fxa/packages/fxa-auth-server/config/dev.json` with the value of your private stripe API key. Ensure the key begins with `pk_test` to guarantee are testing in the correct environment.
+
+Ex:
+
+```
+  "subscriptions": {
+    "enabled": true,
+    "stripeApiKey": "pk_test_####"
+    ...
+```
+
+If you are testing with your own Stripe account, you will need to update the `stripe.apiKey.default` public key set in `fxa/packages/fxa-payments-server/server/config/index.js` to reflect your own. The payments development server _should_ have a default Mozilla public API key in place already but it doesn't hurt to double check.
+
+#### Stripe Product/Plans
+
+To see the available products in the Stripe dashboard, navigate to Billing > Products and click into one of the products to see information about it including the product name, product ID, plan name, plan ID, metadata, logs, and events.
+
+If you are using a new Stripe account, you will need to setup a product and its plan. The product should have additional metadata configured as needed.
+
+##### Product Metadata
+
+| Key                                                   | Value                                                                                                                                                                                           |
+| ----------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| productSet                                            | An arbitrary string used to group products in a set of upgrades & downgrades.                                                                                                                   |
+| productOrder                                          | A number used for sorting products in a set                                                                                                                                                     |
+| webIconURL                                            | Image URL for product icon in web content                                                                                                                                                       |
+| capabilities:{clientID} capabilities:dcdb5ae7add825d2 | Comma-separated list of capabilities enabled by this product for the Relying Party identified by {clientID}                                                                                     |
```

Comment:
There is one more key we support that's worth documenting: `capabilities` without client ID, comma-separated list of capabilities enabled for *every* Relying Party.

(That, or we can discuss removing the feature if it's not actually useful. I don't think it's *used* yet. I can't remember why it's there)